Sources/ActionSheet/ActionSheetCell.swift

```diff
@@ -9,6 +9,58 @@
  * Refer to the COPYING file of the official project for license.
  *****************************************************************************/
 
+enum ActionSheetCellAccessoryType: Equatable {
+    case none
+    case checkmark
+    case disclosureChevron
+    case custom(image: UIImage)
```

Comment:
can we remove the custom? General rule of thumb: Don't try to make something generic before it's needed since it makes code often more complicated